pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -636,4 +636,9 @@ self: super: builtins.intersectAttrs super {
   # need it during the build itself, too.
   cairo = addBuildTool super.cairo self.buildHaskellPackages.gtk2hs-buildtools;
   pango = disableHardening (addBuildTool super.pango self.buildHaskellPackages.gtk2hs-buildtools) ["fortify"];
+
+  # The DateZone xmobar plugin has the hardcoded path /usr/share/zoneinfo
```

Comment:
Thanks for submitting the fix upstream!

Please edit this comment to link to https://github.com/jaor/xmobar/pull/411.